### PR TITLE
update component for textinput on preview

### PR DESCRIPTION
**Pull request by britt6612**
The image on `textinput` preview was not as correct. Originally I wanted to force the focus but that would throw off the accessibility  so I decided to just go with the inline style to match up better with how the focus is in `formfield`

### CODE REVIEW

File: aries-site/src/examples/cardPreviews/textinput.js [hunk 1]
```diff
@@ -4,14 +4,16 @@ import { TextInput, FormField } from 'grommet';
 export const TextInputPreview = () => {
   return (
     <FormField
-      style={{ borderRadius: '4px' }}
-      fill="horizontal"
-      border={{ color: 'focus', size: 'small' }}
+      style={{
+        boxShadow: '0 0 2px 2px #00E8CF',
+        borderRadius: '4px',
+      }}
```
Comment (taysea) [inline aries-site/src/examples/cardPreviews/textinput.js:10]: Think we need htmlFor="focus-id" on the FormField

Comment (britt6612) [reply aries-site/src/examples/cardPreviews/textinput.js:11]: good catch!

Comment (halocline) [reply aries-site/src/examples/cardPreviews/textinput.js:11]: Plus wrap in Form?

Comment (britt6612) [reply aries-site/src/examples/cardPreviews/textinput.js:11]: right! how did I fix all these and miss my own

**Pull request merged**